bundles/org.openhab.automation.pythonscripting/src/main/resources/OH-INF/i18n/pythonscripting.properties

```diff
@@ -23,4 +23,4 @@ automation.config.pythonscripting.injectionEnabled.option.0 = Disable auto injec
 automation.config.pythonscripting.jythonEmulation.label = Enable Jython emulation
 automation.config.pythonscripting.jythonEmulation.description = This enables Jython emulation in GraalPy. It is strongly recommended to update code to GraalPy and Python 3 as the emulation can have performance degradation. For tips and instructions, please refer to <a href="https://www.graalvm.org/latest/reference-manual/python/Modern-Python-on-JVM">Jython Migration Guide</a>.
 automation.config.pythonscripting.scopeEnabled.label = Use scope and import wrapper
-automation.config.pythonscripting.scopeEnabled.description = This enables a scope module and and import wrapper.<br> A scope module is an encapsulated module containing all openHAB jsr223 objects and can be imported with <code>import scope</code><br> Additionally you can run an import like <code>from org.openhab.core import OpenHAB</code>
+automation.config.pythonscripting.scopeEnabled.description = This enables a scope module and and import wrapper.<br> An scope module is an encapsulated module containing all openHAB jsr223 objects and can be imported with <code>import scope</code><br> Additionally, you can run an import like <code>from org.openhab.core import openHAB</code>
```

Comment:
This is not consistent with the new version of the string in the XML. It still has "and and", and bad grammar seems to have been introduced with "An scope module"?